src/app/services/editorial-conventions.service.ts

```diff
@@ -5,7 +5,7 @@ import { EditorialConvention, EditorialConventionLayouts } from '../models/evt-m
 
 // List of handled editorial convention
 export type EditorialConventionDefaults = 'addition' | 'additionAbove' | 'additionBelow' | 'additionInline' | 'additionLeft' | 'additionRight' |
-  'damage' | 'deletion' | 'sicCrux' | 'surplus';
+  'damage' | 'deletion' | 'sicCrux' | 'surplus' | '.sources' | '.analogues' ;
```

Comment:
The dot is a mimic feature for Class (like in CSS), I used it for sources and analogues to show to the user that they are not customizing single elements but group of elements  since many markups (included additions, damages, deletions) can be sources or analogues. Please note that dot in the name is properly managed and doesn't cause any issue.

Reply:
I agree with Chiara. Even if it doesn't cause any issue it's better to use the same notation.